Library/Homebrew/dependency.rb

```diff
@@ -123,11 +121,6 @@ def skip
       throw(:action, :skip)
     end
 
-    # Keep a dependency, but prune its dependencies
-    def keep_but_prune_recursive_deps
```

Comment:
@stek29 Leave all definitions of this method/test as-is: I'll use them on https://github.com/Homebrew/brew/pull/3914.

Reply:
@MikeMcQuaid Sure